docs/ref/forms/api.txt

```diff
@@ -700,6 +700,27 @@ example, in the ``ContactForm`` example, the fields are defined in the order
 ``subject``, ``message``, ``sender``, ``cc_myself``. To reorder the HTML
 output, just change the order in which those fields are listed in the class.
 
+There are three options to specify the order:
+
+.. versionadded:: 1.9
```

Comment:
I would put this annotation below the `.. attribute::` and `.. method:: ` lines.